alacritty/src/logging.rs

```diff
@@ -167,7 +179,12 @@ fn create_log_message(record: &log::Record<'_>, target: &str, start: Instant) ->
 fn is_allowed_target(level: Level, target: &str) -> bool {
     match (level, log::max_level()) {
         (Level::Error, LevelFilter::Trace) | (Level::Warn, LevelFilter::Trace) => true,
-        _ => ALLOWED_TARGETS.contains(&target),
+        _ => {
+            ALLOWED_TARGETS.contains(&target)
+                || EXTRA_LOG_TARGETS
+                    .as_ref()
+                    .map_or(false, |targets| targets.iter().any(|t| t == target))
```

Comment:
`contains` expects `&String` and not `&str`, so we have this mess here.

Reply:
Should just change it to `Vec<String>` instead of `Option<Vec<String>>` by using an empty vec by default? That should simplify things.